bin/cylc-message

```diff
@@ -39,10 +39,11 @@ def main():
     parser = OptionParser(__doc__)
 
     parser.add_option(
-        "-p", "--priority", metavar="PRIORITY", type="choice",
+        "-s", "--severity", "-p", "--priority",
+        metavar="SEVERITY", type="choice",
         choices=['NORMAL', 'WARNING', 'CRITICAL', 'CUSTOM'],
-        help="message priority: NORMAL, WARNING, or CRITICAL; default NORMAL.",
-        action="store", dest="priority", default="NORMAL")
+        help="message severity: NORMAL, WARNING, or CRITICAL; default NORMAL.",
```

Comment:
The help message needs to include `CUSTOM` as well.

Reply:
Done.